src/kusto/HISTORY.rst

```diff
@@ -1,8 +1,8 @@
-.. :changelog:
-
-Release History
-===============
-
-0.1.0
-++++++
-* Initial release.
+.. :changelog:
+
+Release History
+===============
+
+0.1.0
+++++++
+* Initial release.
```

Comment:
It would be better to leave meaningful release notes

Reply:
Done, Thanks! I'll add this to our client-generation docs for next time